tests/CarbonInterval/SetStateTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\CarbonInterval;
+
+use Carbon\CarbonInterval;
+use Tests\AbstractTestCase;
+
+class SetStateTest extends AbstractTestCase
+{
+    public function testEvaluatingVarExportReturnsCarbonIntervalInstance()
+    {
+        $export = var_export(CarbonInterval::minutes(3), true);
+
+        $this->assertInstanceOfCarbonInterval(eval("return $export;"));
```

Comment:
Maybe we should also check the rebuilt object is the correct interval.
```suggestion
        $export = var_export(CarbonInterval::minutes(3), true);
        $interval = eval("return $export;");

        $this->assertInstanceOfCarbonInterval($interval);
        $this->assertSame('3 minutes', $interval->forHumans());
```